apis/beacon/states/validator_identities.yaml

```
@@ -0,0 +1,69 @@
+post:
+  operationId: "postStateValidatorIdentities"
```

The http method used is a technical detail imo and we don't need it as part of operation id, it might be a bit confusing semantically

```suggestion
  operationId: "getStateValidatorIdentities"
```

---

For context, we use the `operationId` as method in our api client and error messages

---

We do seem to use the HTTP method elsewhere in the operation ID, notably we already have `getStateValidators` and `postStateValidators` in the beacon API.  Given that these methods have different signatures (data in body vs in request)  I'd be quite wary in changing it in this way.

---

I think in this instance i'd prefer the `post`... i do think sometimes we call them more in line with the intent, but it also seems very minor...
This way (post) is arguably the same as the current validators endpoint so there's that...

---

yeah, let's keep it `post` then but should avoid adopting this convention in other routes if possible (outside of validators)